src/lib.rs

```diff
@@ -143,3 +195,12 @@ enum Reactor {
     Handle(reactor::Handle),
     Remote(reactor::Remote),
 }
+
+cfg_if! {
+    if #[cfg(feature = "tls")] {
+        extern crate tokio_tls;
+        pub extern crate native_tls;
```

Comment:
Well we need to expose the `TlsAcceptor` type in the builder for the server options. Another option here though instead of just re-exporting the whole crate would be to just re-export the `TlsAcceptor` type itself.

I was also thinking on the `client` side it could take an option `TlsConnector` if people want to override the default. So we could get away with just re-exporting `TlsConnector` and `TlsAcceptor` from `native_tls` instead of the whole crate; that's probably a better idea.